presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -82,7 +82,7 @@ public BasicQueryInfo(QueryInfo queryInfo)
                 queryInfo.getMemoryPool(),
                 queryInfo.isScheduled(),
                 queryInfo.getSelf(),
-                queryInfo.getQuery(),
+                queryInfo.getQueryAbridged(),
```

Comment:
`BasicQueryInfo#query` is not expected to be abridged

Reply:
https://github.com/prestodb/presto/pull/11329#discussion_r212767435